storybooknative/src/Checkbox/Checkbox.stories.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+/* eslint-disable jsx-a11y/accessible-emoji, import/no-unresolved, import/extensions */
```

Comment:
```suggestion
/* eslint-disable import/no-unresolved, import/extensions */
```
I believe the eslint rule regarding emoji's is not necessary anymore.